comps/third_parties/gpt-sovits/src/start.sh

```diff
@@ -0,0 +1,28 @@
+# Copyright (C) 2025 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+#!/bin/bash
+
+# Download default voice reference
+wget "https://github.com/intel/intel-extension-for-transformers/raw/refs/heads/main/intel_extension_for_transformers/neural_chat/assets/audio/welcome_cn.wav"
+
+# Download models
+MODEL_REPO=https://huggingface.co/lj1995/GPT-SoVITS
+llm_download=${llm_download}
+echo "llm_download: ${llm_download}"
+if [ "$llm_download" = "True" ]; then
+  # clean if exists
+  rm -rf /home/user/GPT-SoVITS/GPT_SoVITS/pretrained_models/*
+
+  echo "Please wait for model download..."
+  git lfs install &&  git clone ${MODEL_REPO} /home/user/pretrained_models
```

Comment:
Why this needs all repo history and branches?

Reply:
good catch. Fixed in https://github.com/opea-project/GenAIComps/pull/1622